x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/exporter/http/HttpExporterIT.java

```diff
@@ -93,9 +98,23 @@
     private final boolean currentLicenseAllowsWatcher = true;
     private final boolean watcherAlreadyExists = randomBoolean();
     private final Environment environment = TestEnvironment.newEnvironment(Settings.builder().put("path.home", createTempDir()).build());
+    private final String userName = "elasticuser";
 
     private MockWebServer webServer;
 
+    private MockSecureSettings mockSecureSettings  = new MockSecureSettings();
+    @Override
+    protected Settings nodeSettings(int nodeOrdinal) {
+
+        Settings.Builder builder = Settings.builder()
+            .put(super.nodeSettings(nodeOrdinal))
+            .put(MonitoringService.INTERVAL.getKey(), MonitoringService.MIN_INTERVAL)
+            // we do this by default in core, but for monitoring this isn't needed and only adds noise.
+            .put("index.lifecycle.history_index_enabled", false)
```

Comment:
This needs to change now with the rename:

```suggestion
            .put("indices.lifecycle.history_index_enabled", false)
```